gapipy/resources/base.py

```diff
@@ -30,14 +30,17 @@ def options(cls, client):
 
     def fetch(self):
         logger.info('Fetching %s/%s', self._resource_name, self.id)
-        self.is_stub = False
 
         # Fetch the resource using the client bound on it, which handles cache get/set.
         resource_obj = getattr(self._client, self._resource_name).get(
             self.id,
-            variation_id=getattr(self, 'variation_id', None))
+            variation_id=getattr(self, 'variation_id', None),
+            httperrors_mapped_to_none=None)
+
         if resource_obj:
```

Comment:
Hm, maybe this `if` should just go away since we do `httperrors_mapped_to_none=None`... I wonder if there are other circumstances under which this `.get(...)` will return None